var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -50,12 +52,28 @@ class Julia(MakefilePackage):
     depends_on(
         "llvm"
         " targets=amdgpu,bpf,nvptx,webassembly"
-        " version_suffix=jl +link_llvm_dylib ~internal_unwind"
+        " version_suffix=jl +link_llvm_dylib"
     )
     depends_on("libuv", when="@:1.7")
     depends_on("libuv-julia@1.42.0", when="@1.8.0:1.8.1")
     depends_on("libuv-julia@1.44.2", when="@1.8.2:")
 
+    # Do not use internal unwind.  We need to use a conflict, because
+    # `internal_unwind` is defined only when `+clang`.
+    conflicts("llvm+clang+internal_unwind")
```

Comment:
Here `llvm+internal_unwind` should really be enough, it's different from the requirment ~internal_unwind in that it doesn't force the variant to exist

Reply:
Fixed, thanks!